src/app/BUILD.gn

```diff
@@ -71,45 +77,33 @@ source_set("revision_info") {
   ]
 }
 
-source_set("app_config") {
-  sources = [ "AppConfig.h" ]
+source_set("path-params") {
+  sources = [
+    "AttributePathParams.h",
+    "EventPathParams.h",
+  ]
```

Comment:
EventPath params includes:
   - app/ConcreteEventPath.h
   - app/util/basic-types.h
   - lib/core/CHIPCore.h

So I wonder if it is ok to just have these source_sets without any dependencies and if we maybe just hide away things instead.

Reply:
Based on my understanding, headers don't get validated within their individual source_sets. As long as the dependencies are in the components that depend on them the build will pass.